collection/Jelke Ludolphij; Tavern of Trinkets.json

```diff
@@ -422,16 +422,57 @@
 			}
 		},
 		{
-			"name": "Twinkle Doll",
+			"name": "Twinkle Doll ",
```

Comment:
Mind removing this trailing space?

Reply:
Eh whatever I'll do it later

Reply:
It was to fix the {@i Twinkle Doll} issue, which is still not fixed.

Reply:
Yeah I saw that convo afterwards. True, it's not fixed _in production_ :wink: